app/models/inventory.rb

```diff
@@ -0,0 +1,7 @@
+class Inventory < ApplicationRecord
+  belongs_to :user
+  has_many :inventory_foods
+  has_many :foods
```

Comment:
- [ ] according to the diagram and the migrations you've set up an `Inventory` model should not have many foods. what you can do is say that it has many foods through the inventory_foods table.
```suggestion
  has_many :foods, through: :inventory_foods
```